packages/node_modules/@webex/webex-core/test/unit/spec/credentials/credentials.js

```diff
@@ -228,6 +228,22 @@ describe('webex-core', () => {
       });
     });
 
+    describe('#extractOrgIdFromUserToken()', () => {
+      let credentials;
+      let webex;
+
+      beforeEach('generate webex and destructure credentials', () => {
+        webex = new MockWebex();
+        credentials = new Credentials(undefined, {parent: webex});
+      });
+
+      it('should return undefined if the provided JWT is not valid', () =>
```

Comment:
I know we have the happy path in the integration tests, but could you add one here as well? 

```js
const jwt = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyLCJyZWFsbSI6Im15LXJlYWxtIn0.U16gzUsaRW1VVikJA2VeXRHPX716tG1_B42oxzy1UMk';
const realm = 'my-realm';

assert.isEqual(credentials.extractOrgIdFromJWT(jwt), realm);
```

Reply:
Implemented.